src/routes/jira/sync/jira-sync-post.ts

```diff
@@ -1,15 +1,19 @@
 import { Subscription } from "models/subscription";
 import * as Sentry from "@sentry/node";
 import { NextFunction, Request, Response } from "express";
-import { findOrStartSync } from "~/src/sync/sync-utils";
+import { findOrStartSync, getCommitSinceDate } from "~/src/sync/sync-utils";
 import { sendAnalytics } from "utils/analytics-client";
 import { AnalyticsEventTypes, AnalyticsTrackEventsEnum, AnalyticsTrackSource } from "interfaces/common";
+import { NumberFlags } from "config/feature-flags";
 
 export const JiraSyncPost = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
+	const { jiraHost } = res.locals;
+
 	const { installationId: gitHubInstallationId, syncType, appId: gitHubAppId } = req.body;
 
 	// A date to start fetching commit history(main and branch) from.
-	const commitsFromDate = req.body.commitsFromDate ? new Date(req.body.commitsFromDate) : undefined;
+	const commitsFromDate = req.body.commitsFromDate ? new Date(req.body.commitsFromDate) : await getDefaultSyncDate(jiraHost);
 	Sentry.setExtra("Body", req.body);
```

Comment:
Do we have any use case where we want to run full backfill? Do we still want a capability to trigger via admin api?

Reply:
Do want to control the backfill data time frame? e.g. Can't run backfill for more than 1 year time frame

Reply:
> Do we have any use case where we want to run full backfill? Do we still want a capability to trigger via admin api?

The backfill via the UI is full currently. Unless I misunderstand your questions?

Reply:
> Do want to control the backfill data time frame? e.g. Can't run backfill for more than 1 year time frame

Not sure, could be a new ticket as this is UX enhancement. This change just keeping as it is ( and change the implicit to use the FF on 6months)

Reply:
> Do we have any use case where we want to run full backfill? Do we still want a capability to trigger via admin api?

https://github.com/atlassian/github-for-jira/blob/main/static/js/jira-configuration.js#L92

Reply:
Any idea when are we using partial sync?

Reply:
1. Some api endpoint can allow us to provide full or partial.
2. First initial sync is partial, but I think that should be equivalent to full.

Reply:
So "partial" is not used by user, but only from admin api.